### PR TITLE
Bring example recipe in line with Grayskull

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge/core I updated the sample recipe to what Grayskull would produce so that we have less divergence there. 

A gray skull output would look like this:

```yaml
{% set name = "simplejson" %}
{% set version = "3.8.2" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/simplejson-{{ version }}.tar.gz
  sha256: d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f

build:
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv
  number: 0

requirements:
  host:
    - python
    - pip
  run:
    - python

test:
  imports:
    - simplejson
  commands:
    - pip check
  requires:
    - pip

about:
  home: http://github.com/simplejson/simplejson
  summary: Simple, fast, extensible JSON encoder/decoder for Python
  license: Apache-2.0
  license_file: LICENSE.txt

extra:
  recipe-maintainers:
    - ADD_YOUR_GITHUB_ID_HERE
```

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -61,23 +65,25 @@ test:
...
   # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
   # Use the SPDX identifier, e.g: GPL-2.0-only instead of GNU General Public License version 2.0
   # See https://spdx.org/licenses/
   license: MIT
-  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
+  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". 
+  # Optional
   license_family: MIT
-  # It is strongly encouraged to include a license file in the package,
+  # It is required to include a license file in the package,
```
Comment (beckermr) [inline recipes/example/meta.yaml:81]: cc @conda-forge/core on this specific change

Comment (BastianZim) [reply recipes/example/meta.yaml:81]: Yes I wasn't sure about this one. We don't have it written down anywhere but it has been de-facto standard now. Might need a CFEP.

Comment (ocefpaf) [reply recipes/example/meta.yaml:81]: I don't think we need a CFEP. I are already enforcing a hard requirement for a license file. This is from old days where so many packages did not have one. I think it is fine to keep the change.

**Comment by beckermr** [comment]
@conda-forge-admin restart ci

**Comment by jakirkham** [comment]
Can add `[skip lint]` or `[lint skip]` to the commit message to ignore the linter

**Review by carterbox — APPROVED** [review]

**Pull request merged**